src/main/resources/project/lib/EC/JBoss.pm

```diff
@@ -89,6 +92,12 @@ sub new {
     }
 
     my $creds = $self->get_credentials();
+
+    if ($creds->{log_level} && is_positive_int($creds->{log_level})) {
```

Comment:
Yes, get_credentials method is being used for configuration data retrieval. This method returns a hashref with all information from configuration.